### PR TITLE
Data release preview

### PR DESCRIPTION
**Pull request by jehangiramjad**
This PR will likely not be merged but is for reviewing the diffs based on place import bug fixes.

I updated the bigquery version and also the bigtable import versions. Please verify the diffs to see if they make sense.

### CODE REVIEW

**Comment by shifucun** [comment]
If the diff look good, we should merge this PR :)

File: internal/server/statvar/golden/search_statvar/crime.json [hunk 1]
```diff
@@ -21759,24 +21777,6 @@
           "dcid": "Victim_Count_CriminalIncidents_BiasMotivationRace_CrimeAgainstSociety_IsHateCrime_AsianAlone"
         }
       ]
-    },
-    {
-      "dcid": "dc/g/HateCrimeIncidents_BiasMotivation-race_CrimeType-CrimeAgainstSociety_TargetedRace-BlackOrAfricanAmericanAlone",
```
Comment (ajaits) [inline internal/server/statvar/golden/search_statvar/crime.json:21764]: There is no update to FBI HateCrime yet, so don't expect these deletions.

Comment (shifucun) [reply internal/server/statvar/golden/search_statvar/crime.json:21764]: This seems to be replaced by addition of other star vars. @chejennifer is this possible due to search results not stable?

Comment (chejennifer) [reply internal/server/statvar/golden/search_statvar/crime.json:21764]: it looks like the stat vars that replaced this are ranked "higher" and looks like they were recently added so should make sense? https://critique.corp.google.com/cl/599147687

**Comment by n-h-diaz** [comment]
thanks Jehangir! I think it looks okay for the place changes at least

File: internal/server/v0/triple/golden/get_triples/observation.json [hunk 2]
```diff
@@ -101,46 +101,5 @@
...
-      "subjectId": "dc/o/w2z8nx9y43k97",
-      "predicate": "observationAbout",
-      "objectId": "Earth",
-      "objectName": "World",
-      "provenanceId": "dc/base/WorldDevelopmentIndicators"
-    },
-    {
-      "subjectId": "dc/o/w2z8nx9y43k97",
-      "predicate": "observationDate",
-      "objectValue": "2006",
```
Comment (ajaits) [inline internal/server/v0/triple/golden/get_triples/observation.json:116]: @n-h-diaz This value is present for 2006:
https://screenshot.googleplex.com/BUKK9wJhuaWn4kr

Comment (shifucun) [reply internal/server/v0/triple/golden/get_triples/observation.json:116]: Is this deletion okay?

**Comment by shifucun** [comment]
@pradh @jehangiramjad if the changes look good, we should submit this PR which can make next week's release much easier.

File: internal/server/v0/triple/golden/get_triples/observation.json (hunk 2 above)
Comment (jehangiramjad) [reply internal/server/v0/triple/golden/get_triples/observation.json:116]: @n-h-diaz what was the resolution here?

Comment (shifucun) [reply internal/server/v0/triple/golden/get_triples/observation.json:116]: This golden file is from BigQuery and this observation misses data there. See the following query:

SELECT * FROM `datcom-store.dc_kg_latest.StatVarObservation` 
WHERE id = "dc/o/w2z8nx9y43k97" or id = "dc/o/88cs3xqnmpp55"

vs

SELECT * FROM `datcom-store.dc_kg_2023_12_08_23_23_15.StatVarObservation` 
WHERE id = "dc/o/w2z8nx9y43k97" or id = "dc/o/88cs3xqnmpp55"

Comment (jehangiramjad) [reply internal/server/v0/triple/golden/get_triples/observation.json:116]: Based on the team chat, the resolution seems to be to update the golden file which is now done. @pradh please confirm that this is what you intended? @ajaits @shifucun @n-h-diaz FYI

File: internal/server/statvar/golden/search_statvar/crime.json (hunk 1 above)
Comment (shifucun) [reply internal/server/statvar/golden/search_statvar/crime.json:21764]: Got it. Thanks for checking!

File: internal/server/v1/triples/golden/bulk_triples_in/result1.json [hunk 3]
```diff
@@ -10366,5 +10366,5 @@
       "node": "dummy"
     }
   ],
-  "nextToken": "H4sIAAAAAAAA/2zOwQrCMAyAYZt1WxY8jAji86h3hYEnQeqIUljb0XUHfWofQVavXj5C/kNCV+K9DUN42t4M3Si9deKpSq9RTo9/jVcMqBgQGLBgQM2AJQNWrLFul4IM2DAg0ZvWhzD7dDtLnIKnrRMzzVGc+HQUH5z1JoVImy6ZZKe0HLqYaM19EIZWsUaV/c1FVmfLbNUqLrHefQrWiHnV5B/oCwAA//8BAAD//0LbYVbaAAAA"
+  "nextToken": "H4sIAAAAAAAA/2zOwQrCMAzGcZd1WxY8jAg+kHpXGHgSpI4ohbUdXXfY28vi1cuPwP+Qjx7EJxfH+HGDHftJBuclUJ3XSa7vf413DFgwIDBgyYCGASsGrNlg020FGbBlQKKV9ue4hPy8SZpjoKMXOy9JvIR8kRC9CzbHRIc+2+zmvD262+TsaxSGrmCDhfq7S9WolVqrjYpqqxPoCwAA//8BAAD//yvpAtjZAAAA"
```
Comment (shifucun) [inline internal/server/v1/triples/golden/bulk_triples_in/result1.json:10369]: For this one diff, can you use this token in the test.go file https://github.com/datacommonsorg/mixer/blob/ab802f7c91e3be0341caa69d72cd4b9ff8191125/internal/server/v1/triples/golden/bulk_triples_in_test.go#L51 and similar for the one of internal/server/v2/propertyvalues/golden/triple_in/result1.json

**Review by shifucun — APPROVED** [review]

File: internal/server/v1/triples/golden/bulk_triples_in/result1.json (hunk 3 above)
Comment (jehangiramjad) [reply internal/server/v1/triples/golden/bulk_triples_in/result1.json]: Done

File: internal/server/v0/triple/golden/get_triples/observation.json (hunk 2 above)
Comment (pradh) [reply internal/server/v0/triple/golden/get_triples/observation.json:116]: Yes, please update the golden file with `dc/o/em7gh269k7s2`

Comment (pradh) [reply internal/server/v0/triple/golden/get_triples/observation.json:116]: hmm, the test file seems updated, but not seeing additions in this file?

**Review by pradh — APPROVED** [review]
Per @spiekos the chlamydia deletion is expected: https://screenshot.googleplex.com/98JkWhXVJiokvpe

**Comment by jehangiramjad** [comment]
@pradh wrote: hmm, the test file seems updated, but not seeing additions in this file?
Whooops, yes now updated the goldens!

File: internal/server/v0/triple/golden/get_triples/observation.json (hunk 2 above)
Comment (jehangiramjad) [reply internal/server/v0/triple/golden/get_triples/observation.json:116]: Updated!

**Pull request merged**